libs/ui/src/icons.tsx

```diff
@@ -135,7 +135,7 @@ function iconColor(theme: UiTheme, color?: IconColor) {
     danger: colors.dangerAccent,
     inverse: colors.onInverse,
     inversePrimary: colors.inversePrimary,
-    inverseWarning: 'rgb(255 163 84)',
+    inverseWarning: 'rgb(255 163 84)', // TODO-POLLBOOK-MERGE
```

Comment:
I'm happy to take on adding this color to the design system (as we'll need to make sure it meets contrast requirements). Can you assign a follow up to me?

Reply:
https://github.com/votingworks/vxsuite/issues/6167 ty!